cdap-client/src/main/java/co/cask/cdap/client/StreamClient.java

```diff
@@ -85,7 +87,7 @@ public StreamClient(ClientConfig config) {
    * Gets the configuration of a stream.
    *
    * @param streamId ID of the stream
-   * @throws IOException if a network error occurred
+   * @throws IOException             if a network error occurred
```

Comment:
I think when I refactored the code it aligned the descriptions up so that the description of IOException is aligned with the description of StreamNotFoundException